configs/config.yaml

```
@@ -88,4 +88,21 @@ logging:
         timeout: "30s"
         user_agent: "letraz-utils/1.0"
         headers:
-          Content-Type: "application/json" 
\ No newline at end of file
+          Content-Type: "application/json"
+
+# DigitalOcean Spaces configuration for storing resume screenshots
+digitalocean:
+  spaces:
+    bucket_url: ""  # Set via environment variable BUCKET_URL
+    cdn_endpoint: ""  # Set via environment variable BUCKET_CDN_ENDPOINT  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)